app/models/ability.rb

```diff
@@ -0,0 +1,22 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+
+    user ||= User.new
+
+    can :read, [Product]
+    can [:read,:create], Review
```

Comment:
But if logged in user leaves a  comment it should be not verified also. This is confusing. As I remember from task only logged in user can left comment and comment is alwaysnot verified until admin approves it. Could you check it please ?